# A dummy message to request PCL to publish PCL information
int32 state
